src/bundles/pinning.js

```diff
@@ -27,5 +27,18 @@ export default {
       autoUpload: 'PINS_ONLY',
       addedAt: new Date(1592491648691)
     }
+  ]),
+
+  selectAvailablePinningServices: () => ([
+    {
+      name: 'Pinata',
+      icon: 'https://svgshare.com/i/M7y.svg'
+    }, {
+      name: 'Infura',
+      icon: 'https://i.pinimg.com/originals/e2/8e/98/e28e98282e6ace744098d2d25ddee7d2.png'
+    }, {
+      name: 'Eternum',
+      icon: 'https://www.eternum.io/static/images/icons/favicon-32x32.a2341c8ec160.png'
+    }
```

Comment:
Nit: WebUI should not make requests to remote hostnames, we should include these assets in local build so they are loaded from IPFS.

Reply:
These icons will be provided by the API that manages available pinning services in the future, but you think it's worth adding them now to IPFS?

Reply:
Yes, please add them to IPFS. When showing those screens we want to avoid failures like this:

> ![Screenshot_2020-07-03 Settings - IPFS](https://user-images.githubusercontent.com/157609/86485787-ec27ed00-bd59-11ea-9ca3-bb83c454e795.png)

As for image URL, are you suggesting we persist `ApiIcon` under `Pinning.Services` config map?
If so, we should rais that need and discuss that in https://github.com/ipfs/go-ipfs-config/pull/104  (I'd use CID instead of URL).

Reply:
Done 👍